src/yang/ietf-bgp-policy.yang

```diff
@@ -191,6 +191,33 @@ module ietf-bgp-policy {
     container bgp-defined-sets {
       description
         "BGP-related set definitions for policy match conditions.";
+      container as-path-sets {
+        description
+          "Enclosing container for list of define AS path sets.";
+        list as-path-set {
+          key "name";
+          description
+            "List of defined AS path sets.";
+          leaf name {
+            type string;
+            description
+              "Name of the AS path set -- this is used to reference
+               the set in match conditions.";
+          }
+          leaf-list member {
+            type string;
+            description
+              "AS path regular expression using BGP YANG AS_PATH
+               regular expression syntax.  If any of the regular
+               expressions in the lists are matched, the as-path-set
+               is considered matched.";
+            reference
+              "RFC XXXX: BGP YANG Model for Service Provider
```

Comment:
Please update the reference to the latest title of the draft.

Reply:
If you're going to insist on me cleaning up every bit of dangling reference and punctuation cleanup whenever I try to do structural cleanup, I'm happy to stop doing this work until the rest of the "authors" do the tasks assigned in the other cleanup issues.

Reply:
Addressed in other PR. Thanks.